### PR TITLE
v2.6.3-rc1

### PR DESCRIPTION
**Pull request by brendarearden**
Cherry picked commit from https://github.com/rancher/rancher/pull/35580
Re-ran `go mod tidy` and `go genearte`

### CODE REVIEW

File: package/Dockerfile [hunk 1]
```diff
@@ -110,8 +110,8 @@ RUN curl -sLf ${!TINI_URL} > /usr/bin/tini && \
     chmod +x /usr/bin/tini /usr/bin/telemetry /usr/bin/k3s /usr/bin/kubectl && \
     mkdir -p /var/lib/rancher-data/driver-metadata
 
-ENV CATTLE_UI_VERSION 2.6.1
-ENV CATTLE_DASHBOARD_UI_VERSION v2.6.1
+ENV CATTLE_UI_VERSION 2.6.1-rc1
+ENV CATTLE_DASHBOARD_UI_VERSION v2.6.1-rc1
```
Comment (thedadams) [inline package/Dockerfile:114]: Should these be 2.6.3-rc1?

**Review by thedadams — CHANGES_REQUESTED** [review]

File: package/Dockerfile (hunk 1 above)
Comment (brendarearden) [reply package/Dockerfile]: Yes you are correct. I have updated those with my most recent changes.

**Review by thedadams — DISMISSED** [review]

**Review by kinarashah — DISMISSED** [review]

**Comment by ryansann** [comment]
Per our convo offline, we do need to change kdm branch to `dev-v2.6-1.22`

**Review by ryansann — APPROVED** [review]
This lgtm

**Review by kinarashah — APPROVED** [review]

**Pull request merged**